tai.h

```diff
@@ -0,0 +1,28 @@
+/* Public domain. */
```

Comment:
Can we replace these with the equivalent SPDX line? Would help any tooling that may eventually look at the license of the files.